webapp/views/launch/script.js

```diff
@@ -434,7 +432,7 @@ function handleResponse(res) {
             inputs = $(`.${messageId}-input`)
           }),
           buttonElement('−').click(() => {
-            if (lists.children().length > 4) {
+            if (lists.children().length > 3) {
               lists.children().last().remove()
               lists.children().last().remove()
             }
```

Comment:
It would be nice if you could remove any item, not only the last

Reply:
fixed here 94bddaef53acff5ecaa51e844ea69ba871ba5368